packages/core/src/__tests__/ceramic.test.ts

```diff
@@ -104,7 +100,7 @@ describe('Ceramic integration', () => {
     const doctype1 = await ceramic1.createDocument(DOCTYPE_TILE, { content: { test: 123 } }, { anchor: false, publish: false })
     const doctype2 = await ceramic2.loadDocument(doctype1.id)
     expect(doctype1.content).toEqual(doctype2.content)
-    expectEqualStates(doctype1.state, doctype2.state)
+    expect(DoctypeUtils.statesEqual(doctype1.state, doctype2.state));
```

Comment:
This does basically the same thing as what was there previously, but loses the ability to see the diff between expected and received if the check fails.  Would prefer to leave it as it was

Reply:
okay. changed back in su/state-refactor-tasks-execution